### ISSUE TITLE
fail to reproduce ICDAR2015 result 

### ISSUE THREAD

**Issue by D641593** [ying09/TextFuseNet.pytorch#111]
Hi, thank you for a great work.

I'm having some trouble reproducing the ICDAR2015 result.

First, I use the synth-text pretrained weight to generate character labels.

According to #99 , I fill the boxes with confidence scores > 0.8 to generate masks.

Then, I start training using synth-text pretrained weights and the generated COCO format label.

The result of the model is 
{"precision": 0.7594320486815416, "recall": 0.9012999518536351, "hmean": 0.8243064729194188}.

Although Recall is very close to the original data on paper, Precision drops a lot.

Could anyone give me some help?
